src/chat-messages.js

```diff
@@ -0,0 +1,547 @@
+/**
+ * File: Chat Messages
+ *
+ * Maintainer:      - Sebastian Kippe <sebastian@kip.pe>
+ * Version:         - 0.5.0
+ *
+ * This module stores chat messages in daily archive files.
```

Comment:
Is this file supposed to cover all forms of chat messages? Or just IRC? It seems specific to IRC from a cursory glance, ie. just channel names and stuff.